docker-compose.yml

```diff
@@ -0,0 +1,42 @@
+services:
+  mysqldb:
+    platform: linux/amd64/v3
+    image: mysql
+    restart: unless-stopped
+    env_file: ./.env
+    environment:
+      - MYSQL_ROOT_PASSWORD=$MYSQL_ROOT_PASSWORD
+      - MYSQL_DATABASE=$MYSQLDB_DATABASE
+    ports:
+      - "$MYSQLDB_LOCAL_PORT:$MYSQLDB_DOCKER_PORT"
+    command: --init-file /data/application/init.sql
+    volumes:
+        - ./init.sql:/data/application/init.sql
+  app:
+    depends_on:
+    - mysqldb
+    restart: on-failure
+    image: posts-service
+    build: .
+    env_file: ./.env
+    ports:
+      - "$SPRING_LOCAL_PORT:$SPRING_DOCKER_PORT"
+      - "$DEBUG_PORT:$DEBUG_PORT"
+    environment:
+      SPRING_APPLICATION_JSON: '{
+        "spring.application.name": "online_book_shop",
+        "spring.datasource.url": "jdbc:mysql://mysqldb:$MYSQLDB_DOCKER_PORT/$MYSQLDB_DATABASE",
+        "spring.datasource.driverClassName": "com.mysql.cj.jdbc.Driver",
```

Comment:
You actually can use only in SPRING_APPLICATION_JSON
"spring.datasource.url" 
"spring.datasource.username"
"spring.datasource.password"
"spring.jpa.hibernate.ddl-auto"

Reply:
If not all of the props are here, like JWT staff, how it'll work otherwise?